12-offer-encoding.md

```diff
@@ -229,6 +229,88 @@ The human-readable prefix for offers is `lno`.
     1. type: 22 (`offer_issuer_id`)
     2. data:
         * [`point`:`id`]
+    1. type: 24 (`offer_recurrence`)
```

Comment:
I think we should use odd TLVs so that a payer that doesn't support recurrence can pay for one period.

Reply:
I wondered about that!  I think that may be worse UX, depending on the use case?  They will think it's a one-off payment ("hey that's cheap!") and then get upset when it's not.

Reply:
Ok, now I'm using both.  That way the user can decide...